releases/3/kafka-k8s/bundle.yaml

```diff
@@ -0,0 +1,34 @@
+description: A fast, secure and fault-tolerant Apache Kafka on Kubernetes, supported by Apache ZooKeeper
+issues: https://github.com/canonical/kafka-k8s-bundle/issues/new
+source: https://github.com/canonical/kafka-k8s-bundle
+bundle: kubernetes
+name: kafka-k8s-bundle
+applications:
+  kafka-k8s:
+    charm: kafka-k8s
+    scale: 1
+    channel: 3/edge
+    revision: 39
+    constraints: arch=amd64
+  tls-certificates-operator:
+    charm: tls-certificates-operator
+    scale: 1
+    options:
+      ca-common-name: canonical
+      generate-self-signed-certificates: true
+    channel: latest/edge
```

Comment:
Shouldn't TLS here be `stable` as well?

Reply:
Uhm, not sure. I mean, here it does not matter too much as we specify the `revision` below, that keeps being updated by the automation workflow. That's probably why, although here we say channel `latest/edge`, this did not break (as it is still on revision 27 - which is not however the latest/stable anymore)